internal/operands/tekton-tasks/tekton-tasks.go

```diff
@@ -215,7 +227,7 @@ func (t *tektonTasks) Cleanup(request *common.Request) ([]common.CleanupResult,
 func listDeprecatedClusterTasks(request *common.Request) ([]pipeline.ClusterTask, error) {
 	deprecatedClusterTasks := &pipeline.ClusterTaskList{}
 	err := request.Client.List(request.Context, deprecatedClusterTasks, &client.MatchingLabels{
-		common.AppKubernetesManagedByLabel: common.AppKubernetesManagedByValue,
+		common.AppKubernetesManagedByLabel: common.TektonAppKubernetesManagedByValue,
```

Comment:
If it makes sense to check for both values, please use `In` operation on the label selector:
```go
	labelRequirement, err := labels.NewRequirement(common.AppKubernetesManagedByLabel,
		selection.In, []string{common.TektonAppKubernetesManagedByValue, common.AppKubernetesManagedByValue})
	if err != nil {
		panic(err)
	}

	err = request.Client.List(request.Context, deprecatedClusterTasks, &client.MatchingLabelsSelector{
		Selector: labels.NewSelector().Add(*labelRequirement),
	})
```